apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-examples/platform-action-bar-with-long-title-example.component.html

```diff
@@ -1,18 +1,13 @@
-<fdp-action-bar
-    [contentDensity]="'compact'"
-    [title]="
+<fdp-action-bar [contentDensity]="'compact'" [title]="
         'A Very Long Page Title Which Should Be Truncated at Some Point with a Small Enough Page Width and a Long Enough Text String'
-    "
-    [showBackButton]="true"
-    [description]="
+    " [showBackButton]="true" [description]="
         'This Action Bar description should be composed with enough text to span the entire width of the Action Bar plus a good amount more such that it will force the Action Bar to truncate the text to a more managable size but not so much that it looses its meaning.'

```

Comment:
more than 120 chars in a line, doesn't this throw a lint error?

Reply:
no lint errors for it thrown.